<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeTool</class>
 <widget class="QWidget" name="VolumeTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>80</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="volumeWidget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>60</width>
     <height>300</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#volumeWidget
{
	background-color:rgb(255,255,255);
	border-radius:7px;
}</string>
   </property>
   <widget class="QPushButton" name="silenceBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>255</y>
      <width>60</width>
      <height>45</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>60</width>
      <height>45</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>60</width>
      <height>45</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">#silenceBtn
{
	border:none;
}
#silenceBtn:hover
{
	background-color:rgb(234,234,234);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="volumeRatio">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>225</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>50%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="volumeBox" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>60</width>
      <height>225</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QFrame" name="inLine">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>25</y>
       <width>4</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#inLine
{
	background-color:rgb(236,236,236);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="outLine">
     <property name="geometry">
      <rect>
       <x>28</x>
       <y>25</y>
       <width>4</width>
       <height>180</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#outLine
{
	background-color:rgb(0,204,101);
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="silderBtn">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>18</y>
       <width>14</width>
       <height>14</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#silderBtn
{
	background-color:rgb(0,204,101);
	border-radius:7px;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
